performance/images.md

```diff
@@ -28,48 +49,41 @@ When encoding JPEG images:
+* Making sure that you save images using an [sRGB](https://en.wikipedia.org/wiki/SRGB) ICC profile. Although modern browsers support images with ICCv2 profiles like [Adobe RGB](https://en.wikipedia.org/wiki/Adobe_RGB_color_space) and even [ProPhoto RGB](https://en.wikipedia.org/wiki/ProPhoto_RGB_color_space), colour space issues are extremely difficult to debug. Saving the images as sRGB and removing the colour profile is encouraged.
+* Stripping the metadata from the image. This includes things like EXIF data, Copyright information (unless required), GPS data, colour profiles, thumbnails (in JPEGs), comments, and others.
+* Setting appropriate cache headers for the images. (See [Serving images](#serving-images), below.)
 
-There's also an [ImageOptim-CLI](https://github.com/JamieMason/ImageOptim-CLI) that can be integrated with CI systems.
+For images that are going to be included in a repository, tools like [ImageOptim](https://imageoptim.com/mac) can be used to optimise them and strip the metadata before committing them to your codebase. There's also an [ImageOptim-CLI](https://github.com/JamieMason/ImageOptim-CLI) that can be integrated with CI systems.
 
-If an image is still too big, you can try several other optimisations that will require editing the image. These will allow you to generate images with a smaller size.
+If an image is still too big after following all these optimisations, other options can be used that will require editing the image. These can be used to generate images with a much smaller size than otherwise would be possible:
 
-* Reduce noise and other complexity.
-* Decrease export quality.
-* Blur unimportant areas.
+* Reducing noise or eliminating noise and other complexity from the image, which makes it easier to compress.
+* Blurring unimportant areas will also make the image easier to compress.
+* Finally, decreasing the export quality may be required if the image is still too big.
 
 ## Serving images
 
-We work under the assumption that images are encoded only once but decoded many times. Therefore, encoding times are usually secondary to the savings in e.g. file size.
+We work under the assumption that images are encoded only once but decoded many times. Therefore, increased encoding times, while undesirable, are usually secondary to other concerns like making the image easier to decompress, or having a smaller file size.
 
 When optimising non-content images that are going to be committed to a repo, make sure you always use the maximum settings if you're using lossless optimisations.
 
-For content images, we encourage using an image server like [Immagine](https://github.com/springernature/immagine) or an image service like [Kraken.io](https://kraken.io/) in order to:
+For content images, we encourage using an on-premise or cloud-based image server in order to:
```

Comment:
should we mention Spindoctor here? or is it too platform-specific?

Reply:
I wanted to mention the quality settings from Spindoctor 😬 but this could also be something to reference about the quality setting.

From the Spindoctor README:
>By default, PNGs will be output at compression quality of 95, and JPEGs at 90 (for lossless sources such as PNG or TIFF) or 80 (for lossy sources).

Those seem quite high? This is just a feeling and I don't know how those original settings came about. I suppose with photographic images you could get away with lower but it may be more noticeable against figure images with graphs.

Reply:
Spindoctor is not public or open source, so I thought it might be confusing for external readers to see a product mentioned that is not publicly available. I also don't feel like we should endorse any products, which is why I remove the reference to specific services.

> Those seem quite high? This is just a feeling and I don't know how those original settings came about. I suppose with photographic images you could get away with lower but it may be more noticeable against figure images with graphs.

Correct. Defaults were adjusted in April 2020. Happy to discuss specifics in Slack.